src/applications/mhv-secure-messaging/actions/folders.js

```diff
@@ -91,6 +91,8 @@ export const newFolder = folderName => async dispatch => {
       type: Actions.Folder.CREATE,
       response,
     });
+    dispatch(getFolders());

```

Comment:
I am wondering if this is doing the same thing? 
![image](https://github.com/user-attachments/assets/c2ea10a6-3a6c-4cd4-885b-bc0d03561653)